.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I think you need to have some more things in your .gitignore, or you'll have a bunch of unnecessary files! `.DS_Store`, for one.